detection-rules/brand_impersonation_vanguard.yml

```
@@ -0,0 +1,96 @@
+name: "Brand Impersonation: Vanguard"
+description: "Detects inbound messages from senders using Vanguard-like display names or domains, excluding legitimate Vanguard domains and authenticated communications. Additional checks ensure the sender is not from trusted organizational domains or high-trust sender domains with proper authentication."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    // display name contains Vanguard
+    (
+      strings.ilike(strings.replace_confusables(sender.display_name),
+                    '*vanguard*'
+      )
+      // levenshtein distance similar to Vanguard
+      or strings.ilevenshtein(strings.replace_confusables(sender.display_name),
+                              'vanguard'
+      ) <= 1
+      // sender domain contains Vanguard
+      or strings.ilike(strings.replace_confusables(sender.email.domain.domain),
+                       '*vanguard*'
+      )
+    )
+  )
+  and (
+    (
+      any(beta.ml_topic(body.current_thread.text).topics,
+          .name in (
+            "Security and Authentication",
+            "Secure Message",
+            "Financial Communications"
+          )
+          and .confidence == "high"
+      )
+      or any(beta.ml_topic(beta.ocr(beta.message_screenshot()).text).topics,
+             .name in (
+               "Security and Authentication",
+               "Secure Message",
+               "Financial Communications"
+             )
+             and .confidence == "high"
+      )
+    )
+    and (
+      any(ml.nlu_classifier(body.current_thread.text).intents,
+          .name == "cred_theft" and .confidence == "high"
+      )
+      or any(ml.nlu_classifier(beta.ocr(beta.message_screenshot()).text).intents,
+             .name == "cred_theft" and .confidence == "high"
+      )
+    )
+  )
+  
+  // and the sender is not in org_domains or from Vanguard domains and passes auth
+  and not (
+    sender.email.domain.root_domain in $org_domains
+    or (
+      sender.email.domain.root_domain in (
+        "vanguard.com",
+        "vanguardcharitable.org", // philanthropic giving arm
+        "vanguardmexico.com",
+        "vanguardcanada.ca",
+        "vanguard.co.uk",
+        "vanguard.com.au",
+        "vanguard.com.hk",
+        "vanguardretirement-mail.com",
```

https://platform.sublime.security/messages/e736c8ce1a250d1de7bb64f0b87c56c13ee369c371caef11795ba8af3e5a6418?preview_id=0195d951-a109-70ba-8069-9dd4bdc0b0d6

```suggestion
        "vanguardinvestor.co.uk",
        "vanguardretirement-mail.com",
```